src/components/services/ServiceBlock.jsx

```diff
@@ -1,16 +1,189 @@
+            }
+        )
+        const currentFilterComp = currentFilterArr.map((el) => {
+                return(<ServiceItem data = {el}/>)
+            }
+        )
+        changeService(currentFilterComp)
+    }
+
+    function showMore () {
+        changeService(allServiceList)
+        setMore("hidden")
+    }
 
-const ServiceBlock = () => {
     return (
-        <section id="serviceBlock">
-            <ServiceFilter/>
-            <Service/>
-        </section>
+        <>
+            <section id="serviceBlock">
+                <div id="serviceFilter">
+                    <div className="sectionLeft">
+                        <h1 className="mainHeading">
+                            Услуги
+                        </h1>
+                        <div className="sectionDesc" id="filterDesc">
+                            Для поддержания кожи в отличном состоянии, нужно не забывать регулярно уделять ей внимание. Бесплатная консультация при покупке любой процедры.
+                        </div>
+                    </div>
+                    <div id="filterSect">
+                        <h3>
+                            Фильтры:
+                        </h3>
+                        <ul>
+                            <li id="cosmetic" className={chosenFilterClass[0]} onClick={setFilter}>Косметология</li>
+                            <li id="laser" className={chosenFilterClass[1]} onClick={setFilter}>Лазерная эпиляция</li>
+                            <li id="figure" className={chosenFilterClass[2]} onClick={setFilter}>Коррекция фигуры</li>
+                        </ul>
+                    </div>
+                </div>
+                <div className="serviceList">
+                    {shownServices}
+                </div>
+            </section>
+            <div className={moreClass} onClick={showMore}>
+                <div>Смотреть ещё</div>
+                <div id="downArr"></div>
+            </div>
+            <section id="mServiceBlock">
+                <h1 className="mainHeading">
+                    Услуги
+                </h1>
+                <div className="mInnerSec">
+                    <div className="headSec">
+                        Косметология
+                    </div>
+                    <div className="mSecDesc">
+                        Для поддержания кожи в отличном состоянии, нужно не забывать регулярно уделять ей внимание.
+                    </div>
+                    <div className="mSecImg" id="cosmeticImg">
+
+                    </div>
+                    <Accordion defaultActiveKey="1" flush>
+                        <Accordion.Item eventKey="0">
+                            <Accordion.Header className="moreMobile">Смотреть список услуг</Accordion.Header>
+                            <Accordion.Body>
+                                  <div className="mobileListItem">
+                                      <div className="mItemHead">Комбинированная чистка</div>
```

Comment:
If we have all info in text.js, why do we need to duplicate it here?
We can just write smth like below:
`pageText().filter(p => p.type == 'cosmetic').map(p => {
     return   <div className="mobileListItem">
                                      <div className="mItemHead">{p.heading}</div>
                                      <div className="mItemPrice">{p.price}</div>
                  </div>
})`

Don't forget that pageText shouldn't be a function